wasm/wat/names_test.go

```diff
@@ -10,38 +10,38 @@ func TestEncodeNameSection(t *testing.T) {
 	m, err := parseModule(simpleExample)
```

Comment:
Maybe we should add case where module name doesn’t exist but function names exist in thhe custom section?

Reply:
Actually that happens for TinyGo produce such section and it is of course valid!

Reply:
oh cool. will do!